test/installer/test_pip_installer.py

```diff
@@ -50,6 +50,11 @@ def test_install(check_output, check_call):
 
         args = args_list[1][0][0]
         assert args[0] == python_path
+        assert args[1:-1] == [
```

Comment:
Is it just me or does this test seems like a [Change Detector](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) - sure we need to fix the existing portion, but is baking in even more change detectors actually gaining us any stability in this tool?